group_vars/logs_library/ssh_client_settings.yml.new

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
This file should not have been committed and content looks the same as `group_vars/logs_library/ssh_client_settings.yml` at first glance.

Reply:
Correct - thanks.